src/components/sections/Footer.jsx

```diff
@@ -0,0 +1,23 @@
+const Footer = () => {
+    return (
+        <div className="bg-black text-white">
+            <div className="py-16 flex items-center justify-center mx-2 ">
+                <div className="flex gap-2 flex-col pr-10 border-r border-solid">
+                    <h1 className="font-bold text-lg">The news</h1>
+                    <p>Kigali, Rwanda</p>
+                    <p>Email: mhenriette00@gmail.com</p>
```

Comment:
Try to wrap  this email in a link or a tag, so that when it's clicked it opens the email box